tests/show_techsupport/test_techsupport.py

```diff
@@ -0,0 +1,515 @@
+    :param duthost: DUT object
+    :param acl_table_config: ACL table configuration dictionary
+    :return: forwards acl_table_config
+    """
+
+    name = acl_table_config['name']
+    conf = acl_table_config['config_file']
+    loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix='acl_rules')
+    loganalyzer.load_common_config()
+
+    try:
+        pdb.set_trace()
+        loganalyzer.expect_regex = [LOG_EXPECT_ACL_TABLE_CREATE_RE]
+        with loganalyzer:
+            logger.info('creating ACL table: applying {}'.format(conf))
+            duthost.command('sonic-cfggen -j {} --write-to-db'.format(conf))
+    except LogAnalyzerError as err:
+        # cleanup config DB if create failed
+        duthost.command('config acl remove table {}'.format(name))
+        raise err
+
+    try:
+        yield acl_table_config
+    finally:
+        loganalyzer.expect_regex = [LOG_EXPECT_ACL_TABLE_REMOVE_RE]
+        with loganalyzer:
+            logger.info('removing ACL table {}'.format(name))
+
+            duthost.command('config acl remove table {}'.format(name))
+
+        # save cleaned configuration
+        duthost.command('config save -y')
+
+
+def teardown_acl(dut, acl_setup):
+    """
+    teardown ACL rules after test by applying empty configuration
+    :param dut: DUT host object
+    :param setup: setup information
+    :return:
+    """
+    pdb.set_trace()
+    dst = DUT_TMP_DIR
+    logger.info('removing all ACL rules')
+    # copy rules remove configuration
+    dut.copy(src=os.path.join(FILES_DIR, ACL_REMOVE_RULES_FILE), dest=dst)
+    remove_rules_dut_path = os.path.join(dst, ACL_REMOVE_RULES_FILE)
+    # remove rules
+    logger.info('applying {}'.format(remove_rules_dut_path))
+    dut.command('config acl update full {}'.format(remove_rules_dut_path))
+
+
+@pytest.fixture(scope='function')
+def acl(duthost, acl_setup, acl_table):
+    """
+    setup rules on DUT
+        :param dut: dut host
+        :param acl_setup: setup information
+        :param acl_table: acl table creating fixture
+        :param acl_rules acl rules setup fixture
+        :return:
+    """
+
+    name = acl_table['name']
+    dut_conf_file_path = os.path.join(acl_setup['dut_tmp_dir'], 'acl_rules_{}.json'.format(name))
+
+    logger.info('generating config for ACL rules, ACL table {}'.format(name))
+    duthost.template(src=os.path.join(TEMPLATE_DIR, ACL_RULES_FULL_TEMPLATE),
+                 dest=dut_conf_file_path)
+
+    logger.info('applying {}'.format(dut_conf_file_path))
+    duthost.command('config acl update full {}'.format(dut_conf_file_path))
+
+
+
+
+#########################################
+########### MIRRORING PART ##############
+#########################################
+
+
+MIRROR_RUN_DIR = os.path.join('mirror_tmp', os.path.basename(BASE_DIR))
+ACL_TABLE_NAME = "EVERFLOW"
+DEFAULT_MIRROR_STAGE = "ingress"
+
+LOG_EXCEPT_EVERFLOW_TABLE_CREATE = '.*Create ACL Group member*.'
```

Comment:
I suggest to create a common directory in tests/, e.g.: tests/logmessages.py and put messages there

Reply:
done